cloudhub-2/modules/ROOT/pages/ch2-ha-dr.adoc

```diff
@@ -0,0 +1,210 @@
+
+==== Licensing
+
+* For organizations using UBP, this configuration doesn’t consume messages or throughput. The configuration consumes fewer flows if you configure the application with xref:ch2-configure-horizontal-autoscaling.adoc[CPU-based HPA].
+* For organizations not using UBP, this configuration consumes lower vCores if the backup deployment is in a scaled-down state before it scales up in response to an outage in the primary region.
+
+==== Recovery Time Objective
+
+If the backup application is in a fully scaled state, you experience lower disruption in the event of an outage. If the backup application is in a scaled-down state, the RTO depends on the time it takes to scale the application up either through control-plane-triggered redeployments, or through xref:ch2-configure-horizontal-autoscaling.adoc[CPU-based HPA].
+
+For integrating systems to resend events or data of in-flight transactions, capture recovery points in logs and monitoring. This builds reliability in your applications.
+
+=== Use Case 3: Cold Standby Configuration
+
+==== When to Use
+
+Low-impact, stateless, or stateful applications that tolerate longer downtime during recovery.
+
+==== Application State
+
+* Set up the infrastructure, but configure your application to a stopped state in the passive region.
+* Start the application only if you detect an outage.
+
+==== Traffic Management
+
+* Use a BYO global load balancer to route traffic only to the region that is up.
+* Perform health checks to determine DR, and scripts to start your applications in backup state and switch traffic routing.
+* If the control plane is unavailable, you can't start the backup application in the runtime plane.
+
+==== Licensing
+
+Because the backup application is in a stopped state, this configuration doesn't consume additional vCores, Mule flows, Mule messages or Data throughput until the backup application starts in the event of a disaster.
+
+==== Recovery Time Objective
+
+The RTO depends on the time it takes to start applications via the control plane.
+
+== Runtime Plane High Availability and Disaster Recovery Summary
+
+
+High-Impact Stateless API:: Use an active-active setup with active replicas in two regions and an external load balancer.
+
+High-Impact Stateful API:: Use an external load balancer to route traffic, and perform health checks to determine DR and switch traffic.
+
+Medium-Impact Stateless and Stateful API::
++
+* Use cold/warm standby, with applications deployed to both regions.
+* Stop or scale down the applications or replicas in the secondary region.
+
+
+== CloudHub 2.0 High Availability and Disaster Recovery Improvements
+
+CloudHub 2.0 improvements over CloudHub include:
+
+* The runtime plane region and the control plane region are loosely coupled due to a service-oriented architecture.
```

Comment:
The sentence structure of each bullet should flow from the intro. For example:

"Loose coupling of the runtime plane region and control plane region due to a service-oriented architecture."